buildkite/src/Command/Base.dhall

```diff
@@ -72,7 +82,31 @@ let build : Config.Type -> B/Command.Type = \(c : Config.Type) ->
     key = Some c.key,
     label = Some c.label,
     plugins =
-      Optional/map Docker.Type B/Plugins (\(docker: Docker.Type) -> B/Plugins.Plugins/Type (toMap { `docker#v3.5.0` = docker })) c.docker
+      let dockerPart =
+        Optional/toList
+          (Map.Type Text Plugins)
+          (Optional/map
+            Docker.Type
+            (Map.Type Text Plugins)
+            (\(docker: Docker.Type) ->
+              toMap { `docker#v3.5.0` = Plugins.Docker docker })
+            c.docker)
+      let summonPart =
+        Optional/toList
+          (Map.Type Text Plugins)
+          (Optional/map
+            Summon.Type
+            (Map.Type Text Plugins)
+            (\(summon: Summon.Type) ->
+              toMap { `angaza/summon#v0.1.0` = Plugins.Summon summon })
+            c.summon)
+
+      -- Add more plugins here as needed, empty list omits that part from the
+      -- plugins map
+      let allPlugins =
+        List/concat (Map.Entry Text Plugins) (dockerPart # summonPart)
+      in
+      Some (B/Plugins.Plugins/Type allPlugins)
```

Comment:
Since all are optional, maybe we should do something like:

```dhall
if List.null (Map.Entry Text Plugins) allPlugins then
  None B/Plugins
else
  Some (B/Plugins.Plugins/Type allPlugins)
```

This way we'll omit the plugins altogether if the list is empty